x/airdrop/types/genesis_test.go

```diff
@@ -0,0 +1,42 @@
+package types_test
```

Comment:
you can scratch this for now until we implement validate genesis (since we'll probably use a different setup for that than what ignite used)